### PR TITLE
Feature/fix mobile nav

### PR DESCRIPTION
**Pull request by im-fdavies**
add mobile nav to fix page layout issues

### CODE REVIEW

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -1,63 +1,90 @@
...
+class Navbar extends Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			view: false
+		};
+	}
+
+	render() {
+		console.log(this.state);
```
Comment (bnjn) [inline src/components/Navbar.jsx]: remove for production

File: src/styles/navbar-component.module.css [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+    margin-top: 0 !important;
+    padding-bottom: 0 !important;
+    border-left: 3px solid white;
+    font-size: 1em !important;
+    text-align: left;
+    z-index: 10000;
+}
+
+.submenu:hover, ul:hover {
+    display: flex;
```
Comment (bnjn) [inline src/styles/navbar-component.module.css:85]: Causes sub links to display on the same line when clicked or hovered over. Removing this line doesn't appear to break anything but fixes the issue.

**Review by bnjn — CHANGES_REQUESTED** [review]
One bug to fix (see code comments)

File: src/styles/navbar-component.module.css (hunk 2 above)
Comment (bnjn) [reply src/styles/navbar-component.module.css:85]: Tested without this line and can't make it break. Will remove in next commit.

**Review by bnjn — APPROVED** [review]
Fixed sub item bug. Also sorted the borked contact form display on mobile.

**Review by bnjn — APPROVED** [review]

File: src/components/Navbar.jsx (hunk 1 above)
Comment (shik4rishuffle) [reply src/components/Navbar.jsx]: good spot

**Comment by shik4rishuffle** [comment]
> Fixed sub item bug. Also sorted the borked contact form display on mobile.

champion

**Pull request merged**